internal/zinc-persist/src/main/scala/sbt/internal/inc/converters/InternalApiProxy.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Zinc - The incremental compiler for Scala.
+ * Copyright 2011 - 2017, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * This software is released under the terms written in LICENSE.
+ */
+
+package xsbti.api
+
+/**
+ * Defines a proxy to the Java compiler interface to create different utils.
+ *
+ * This proxy is required for an efficient deserialization of the analysis files.
+ * It exposes implementation details and uses protected methods to create new
+ * instances of other classes.
+ *
+ * Even though this proxy is public, Do not depend on it, it has no binary compatibility
+ * guarantee and can be broken in any minor release.
```

Comment:
Can we move that to the `internal` namespace? Also, the path doesn't match the package name currently.